### ISSUE TITLE
Exporting and overwriting an existing .json file doesn't show the new map data until the file has been reopened inside the editor

### ISSUE THREAD

**Issue by mouzedrift** [AliveTeam/qt-editor#30]
https://user-images.githubusercontent.com/43358824/144388315-56ea5879-54a0-47b8-952d-5354382cbb0d.mp4

**Comment by paulsapps** [comment]
Is this still an issue?

**Comment by mouzedrift** [comment]
yeah but now you end up with 2 tabs for the same .json since the path is temp until its saved

https://user-images.githubusercontent.com/43358824/146654011-161dfea9-35bb-4720-bad8-d8104fcf530f.mp4

**Comment by paulsapps** [comment]
seems like this is an issue with SaveAs also
